packit_service/service/models.py

```diff
@@ -145,3 +152,61 @@ def create(
         if save:
             b.save()
         return b
+
+
+def get_sa_session() -> Session:
+    """ get SQLAlchemy session """
+    global session_instance
+    if session_instance is None:
+        url = (
+            f"postgres+psycopg2://{os.getenv('POSTGRESQL_USER')}"
+            f":{os.getenv('POSTGRESQL_PASSWORD')}@postgres:5432/{os.getenv('POSTGRESQL_DATABASE')}"
+        )
+        engine = create_engine(url)
+        Session = sessionmaker(bind=engine)
+        session_instance = Session()
+    return session_instance
+
+
+# https://github.com/python/mypy/issues/2477#issuecomment-313984522 ^_^
+if TYPE_CHECKING:
+    Base = object
+else:
+    Base = declarative_base()
+
+
+class GitHubProject(Base):
+    __tablename__ = "github_projects"
+    id = Column(Integer, primary_key=True)
+    namespace = Column(String)
+    repo_name = Column(String)
+
+
+class PullRequest(Base):
+    __tablename__ = "pull_requests"
+    id = Column(Integer, primary_key=True)  # our database PK
+    pr_id = Column(
+        String, index=True
+    )  # GitHub PR ID - let's not make this PK since we can't control it
```

Comment:
:+1:

Reply:
bear in mind that the models are really just a first pass, they definitely need a ton of improvements, that's why I didn't want to commit to create and apply real database schema